umbraco-cloud/deployment/README.md

```diff
@@ -6,29 +6,35 @@ description: >-
 
-![Left to right model](images/left-to-right.png)
+The mainline environments are used when building and deploying the initial website. Upgrades, both manual and automatic also go through the mainline environments.
+
+Flexible environments can be used to work on features separate from the mainline. This is done without interfering with upgrades or other changes being worked on in the mainline.
+
+![Left to right model](images/left-to-right-approach.png)
 
 ## Deployment Approach
 
-Umbraco Cloud separates metadata and content during deployment. Metadata includes Document Types, Templates, Forms, Views, and config files. Content includes content items and media.
+Umbraco Cloud separates schema and content during deployment. Schema includes Document Types, Templates, Forms, Views, and config files. Content includes content items and media.
 
-**Key Terms:**
+* **Deploy:** Moves schema between environments using a Git client or the Umbraco Cloud Portal.
+* **Transfer:** Move content and media directly via the Umbraco backoffice.
 
-* **Deploy:** Moves metadata between environments using a Git client or the Umbraco Cloud Portal.
-* **Transfer:** Moves content and media directly via the Umbraco backoffice.
+### Types of Deployments
 
-**Deployment Types:**
+* **Schema Deployment:** Schema is stored in a Git repository. These are **deployed** between environments using a Git client or the Umbraco Cloud Portal.
+* **Content and Media Transfer:** Content and Media items are not stored in the Git repository. They must be **transferred** directly from the Umbraco backoffice using the **Queue for Transfer** option. Once queued, use the **Deployment** Dashboard in the **Content** section to complete the transfer.
 
-* **Metadata Deployment:** Metadata, including Document Types, Templates, Forms, Views, and config files, is stored in a Git repository. These are **deployed** between environments using a Git client or the Umbraco Cloud Portal.
-* **Content and Media Transfer:** Content and Media items are **not** stored in the Git repository. Instead, they must be **transferred** directly from the Umbraco backoffice using the **Queue for Transfer** option. Once all required items are queued, the **Deployment** Dashboard in the **Content** section is used to complete the transfer.
+| Schema Deployments | Content and Media Transfers |
+| ----------- | ------------- |
+| Schema is stored in a Git repository. These are **deployed** between environments using a Git client or the Umbraco Cloud Portal. | Content and Media items are not stored in the Git repository. They must be **transferred** directly from the Umbraco backoffice using the **Queue for Transfer** option. Once queued, use the **Deployment** Dashboard in the **Content** section to complete the transfer. |
```

Comment:
This is duplicate info. Did you miss removing the bullet points above or this table?